bin/gbuild

```diff
@@ -143,7 +169,7 @@ EOF" if build_desc["sudo"] and @options[:allow_sudo]
     build_desc["remotes"].each do |remote|
       dir = sanitize(remote["dir"], remote["dir"])
 
-      author_date = `cd inputs/#{dir} && git log --format=@%at -1 | date +"%F %T" -u -f -`.strip
+      author_date = `cd inputs/#{dir} > /dev/null && git log --format=@%at -1 | date +"%F %T" -u -f -`.strip
```

Comment:
what is the output that this is suppressing?

Reply:
From what I could figure out, the `cd` was spitting out information about the new directory -- I think in the case where you/your-distro have your shell configured to update the "window title" to reflect the current working directory.  Apparently this isn't an issue on the non-Centos invocations?

Reply:
This shouldn't happen in non-interactive shells.  Perhaps your personal setup has some interactive shell setup or an alias to `cd` in the wrong shell startup file?

Reply:
I tracked it down -- it's because I have `CDPATH` set: https://unix.stackexchange.com/questions/245453/how-to-not-show-path-after-cd-command-with-cdpath-set  (note the workaround).